parsers/runner.py

```diff
@@ -122,7 +124,7 @@ class USTbonds(ParserBase):
     source_url = "https://www.treasury.gov/resource-center/data-chart-center/interest-rates/Pages/TextView.aspx?data=yield"
 
     def all_items(self):
-        return ust.yield_ust_dict(self.start)
+        return ust.get_ust_dict(self.start)
```

Comment:
Switching from iterator to list makes this a little different from other parsers. Can keep the list, but it makes parsers a little different one from another.

Reply:
not sure for now